core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -83,6 +88,7 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
         )
       )
     } else {
+      createdWorkspaceCounter.inc()
```

Comment:
same question as above: do we want to record the metric when we try, vs when the call succeeds. In this case, I think the 2 numbers are most likely pretty similar, so it may not matter much

Reply:
In line with the other discussion I think succeeded calls should be tracked, but for this case I'm not entirely sure where to place the incrementation. I moved the line to `WorkspaxeService.scala` but there's probably a better place for it, if you have any input on it